src/api/readDeviceReferenceData/src/v1/steps.py

```diff
@@ -0,0 +1,63 @@
+from aws_lambda_powertools.utilities.data_classes import APIGatewayProxyEvent
+from domain.core.cpm_product.v1 import CpmProduct
+from domain.core.device_reference_data.v1 import DeviceReferenceData
+from domain.core.product_team.v3 import ProductTeam
+from domain.repository.cpm_product_repository.v3 import CpmProductRepository
+from domain.repository.device_reference_data_repository.v1 import (
+    DeviceReferenceDataRepository,
+)
+from domain.repository.product_team_repository.v2 import ProductTeamRepository
+from domain.request_models.v1 import DeviceReferenceDataPathParams
+from domain.response.validation_errors import mark_validation_errors_as_inbound
+from event.step_chain import StepChain
+
+
+@mark_validation_errors_as_inbound
+def parse_path_params(data, cache) -> DeviceReferenceDataPathParams:
+    event = APIGatewayProxyEvent(data[StepChain.INIT])
+    return DeviceReferenceDataPathParams(**event.path_parameters)
+
+
+def read_product_team(data, cache) -> ProductTeam:
```

Comment:
Do we need this? Can we not do one call to read_product? It should return Not Found if the product team doesn't exist

Reply:
personally i'd prefer a blanket not found and we do a single read